master/internal/authz/obfuscate.go

```diff
@@ -32,6 +32,7 @@ func ObfuscateContainer(container *containerv1.Container) error {
 	}
 	container.Id = hiddenString
 	container.Parent = hiddenString
+	container.PermissionsDisabled = true
```

Comment:
question: Why is this set to true?

Reply:
Since `ObfuscateContainer` gets called when the user is denied viewer permissions for a slot. So it acts as a way of confirming in the API response for `GetAgents` or `GetSlots` that a particular container /task allocation cannot be accessed. Then, in `get_task_name`, we check the permissions for that container and if viewing permissions are disabled for the task running in that container, that's where we use `NO_PERMISSIONS` in replacement of the task name.

Reply:
I think I'm following, it's being used as a flag to mark a container that the user making the request doesn't have access to? Would `PermissionDenied` also work as another name for this field? 

I might be getting caught up on the idea that the permissions are disabled and because of this anyone can view the task name.

Reply:
I see what you're saying. Yea, I'll change the name to `PermissionDenied` so that its clearer!